pkg/reader/ccip.go

```diff
@@ -498,6 +498,9 @@ func (r *ccipChainReader) DiscoverContracts(
 		consts.ContractNameNonceManager: {
 			destChain: staticConfig.NonceManager,
 		},
+		consts.ContractNameRMNRemote: {
+			destChain: staticConfig.RmnProxy,
```

Comment:
I think this is incorrect, its not called `RmnProxy` anymore but just "rmn"

Reply:
https://github.com/smartcontractkit/chainlink/blob/207b0d502a9cadb600873cce9602bdfacca0e177/contracts/src/v0.8/ccip/offRamp/OffRamp.sol#L91

Reply:
Good catch